installing/installing_bare_metal_ipi/ipi-install-overview.adoc

```diff
@@ -6,23 +6,28 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Installer-provisioned installation on bare metal nodes deploys and configures the infrastructure that a {product-title} cluster runs on. This guide provides a methodology to achieving a successful installer-provisioned bare-metal installation. The following diagram illustrates the installation environment in phase 1 of deployment:
+Installer-provisioned installation on bare metal nodes deploys and configures the infrastructure that an {product-title} cluster runs on. This guide provides a methodology to achieving a successful installer-provisioned bare-metal installation. The following diagram illustrates the installation environment in phase 1 of deployment:
 
 image::210_OpenShift_Baremetal_IPI_Deployment_updates_0122_1.png[Deployment phase one]
 
 For the installation, the key elements in the previous diagram are:
 
-- **Provisioner**: A physical machine that runs the installation program and hosts the bootstrap VM that deploys the controller of a new {product-title} cluster.
+- **Provisioner**: A physical machine that runs the installation program and hosts the bootstrap VM that deploys the control plane of a new {product-title} cluster.
 - **Bootstrap VM**: A virtual machine used in the process of deploying an {product-title} cluster.
 - **Network bridges**: The bootstrap VM connects to the bare metal network and to the provisioning network, if present, via network bridges, `eno1` and `eno2`.
+- **API VIP**: An API virtual IP address (VIP) is used to provide failover of the API server across the control plane nodes. The API VIP first resides on the bootstrap VM. `systemd` manages a `keepalived` instance. A script generates the `keepalived.conf` configuration file before launching the service. The VIP moves to one of the control plane nodes after the bootstrap process has completed and the bootstrap VM stops.
```

Comment:
Generally fine, but systemd isn't running keepalived. It's a static pod managed by kubelet and crio.